apps/frontend/src/pages/quiz/libs/components/quiz-form/quiz-form.tsx

```diff
@@ -100,6 +101,7 @@ const QuizForm: React.FC<Properties> = ({ onNext }: Properties) => {
 			if (isLast) {
 				const answerIds = getAnswerIds(data);
 				void dispatch(quizActions.saveAnswers({ answerIds }));
+				void dispatch(authActions.hasQuizAnswer());
```

Comment:
Why do we do that exactly? I hope this won't block the ability to retake the quiz

Reply:
This is just for initial run of the quiz part. if you try to run the quiz route again after finishing it, it still runs the same